client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -366,8 +368,6 @@ export const handlePlatformCheckoutEmailInput = ( field, api ) => {
 		// Insert the wrapper into the DOM.
 		parentDiv.insertBefore( loginSessionIframeWrapper, null );
 
-		setPopoverPosition();
```

Comment:
This unnecessary function here is the main cause of the auto scroll. This function sets the position of the OTP modal. It's not needed here.